leaf/extensions/image.py

```diff
@@ -0,0 +1,220 @@
+import enum
+
+import PIL.Image
```

Comment:
Is PIL not blocking? Also what is the point of importing `PIL.Image` when you've already used `from PIL import Image`?

Reply:
Fixed. And yes, it might be blocking but I think these calls are generally pretty fast so shouldn't actually cause too much of an issue. More expensive commands might cause more of a problem

Reply:
We could run some of these commands in an executor if they become problematic.

Reply:
Yeah as far as I'm aware there's no async version for pillow